maven-core/src/site/apt/artifact-handlers.apt

```diff
@@ -35,7 +35,7 @@ Default Artifact Handlers Reference
 *--------------------+---------------+------------+------------+-----------+---------------------+-----------------------+
 || type              || classifier   || extension || packaging || language || added to classpath || includesDependencies ||
 *--------------------+---------------+------------+------------+-----------+---------------------+-----------------------+
-| <<<pom>>>          |               | <= type>   | <= type>   | none      |                     |                       |
+| <<<pom>>>          |               | <= type>,  Note: this differs for POM files outside the repository which usually use <<<xml>>> as filesystem extension. | <= type>   | none      |                     |                       |
```

Comment:
impossible to understand

Reply:
I would appreciate a more helpful criticism!

Reply:
sorry, we had so many discussions that I don't see how to tell.
as said by Tamasz, there is a mix
In addition, such a text will break the table display.

eventually, if you absolutely want to write something, you can add a note at the end of the page that could say something like:
"Dependency type <<<pom>>> has an extension <<<.pom>>> because POM is stored in repositories as <<<artifactId-version.pom>>>.
During development, as stored in source control, the file is by default named <<<pom.xml>>>"

I don't see how useful it is, but it says what you seem to need to say and does not break the dependency type table